tests/ZiraLink.Api.Application.UnitTests/Services/CustomerServiceTest.cs

```diff
@@ -20,19 +30,122 @@ public CustomerServiceTest()
 
         #region GetCustomerByExternalId
         [Theory]
-        [InlineData("4")]
+        [InlineData("1")]
         public async Task GetCustomerByExternalId_WhenEverythingIsOk_ShouldHasData(string externalId)
         {
             //Arrange
             var mockConfiguration = new Mock<IConfiguration>();
-            mockConfiguration.Setup(m => m["ZIRALINK_URL_IDS"]).Returns("https://ids.ziralink.local:5001");
-   
+            mockConfiguration.Setup(m => m[It.IsAny<string>()]).Returns("https://ids.ziralink.local:5001");
```

Comment:
`ZIRALINK_URL_IDS` should be remained, `It.IsAny<string>()` is not correct

Reply:
Done.